common/src/main/scala/explore/utils/package.scala

```diff
@@ -309,3 +310,16 @@ def keyedSwitchEvalMap[F[_]: Concurrent, I, O, K](
 
 def timeDisplay(name: String, time: TimeSpan, sep: String = ": ") =
   <.span(<.span(ExploreStyles.SequenceTileTitleItem)(name, sep), TimeSpanView(time))
+
+extension (s: NonNegShort)
+  def +|(i: Int): NonNegShort =
```

Comment:
Is this safe? You could add a negative number and get a number less than 0. Similarly, you could subtract a negative number and overflow the short.